doc/introduction/index.rst

```diff
@@ -169,49 +163,45 @@ Industrial Workload Consolidation
    ACRN Industrial Workload Consolidation scenario
 
 Supporting Workload consolidation for industrial applications is even
-more challenging. The ACRN hypervisor needs to run both safety-critical
-and non-safety workloads with no interference, increase security
-functions that safeguard the system, run hard real-time sensitive
-workloads together with general computing workloads, and conduct data
-analytics for timely actions and predictive maintenance.
+more challenging. The ACRN hypervisor needs to run different workloads with no
```

Comment:
change the ``:width: 400px`` to ``:width: 600px`` for the image on line 159 as well